test/document_stores/test_base.py

```diff
@@ -476,6 +513,309 @@ def test_get_all_documents_large_quantities(self, ds):
         assert all(isinstance(d, Document) for d in documents)
         assert len(documents) == len(docs_to_write)
 
+    @pytest.mark.integration
+    def test_multilabel(self, ds):
```

Comment:
I think this is worth splitting into a few separate tests and the labels moved into a fixture. 

It seems like there are a few of these reusing the same list or a very similar one, below this one: they can all be split into smaller chunks ihmo.

Reply:
Agree, the list of labels is very similar but subtly different, I tried to put them in a fixture but tests then fail with non-trivial errors, so I decided to just copy/paste the existing tests.